.custom-format.py

```diff
@@ -19,13 +19,18 @@
 import sys
 import os
 import re
+import shutil
 
 # string_re = re.compile('(\\"(\\\\.|[^"\\\\])*\\")') # future use
 
 with open(".clang-format") as f:
     fmt = f.read()
 
 CLANG_FORMAT_BIN = os.getenv("CLANG_FORMAT_BIN")
+
+if CLANG_FORMAT_BIN is None:
+    CLANG_FORMAT_BIN = shutil.which("clang-format")
+
```

Comment:
the problem here is that clang-format stuff changes with versions. hence if I use it on my system the result will be different to what the container has for testing.
it should be clang-13 IMHO

Reply:
That'll probably make it harder for users to know what to do here, and that opinion might change over time (13 was released just half a year ago). The README also mostly mentions 12 (and 13 on one occurence in the FAQ).

Perhaps correlate it with whatever is the compiler du jour for compiling AFL++? The stable Dockerfile is boasting 14, which is a clear setpoint of what it should be. The README could be generalized with a variable before the install command as in the Dockerfile in this PR.

Then 14 could be the thing now, and it'll be 15 when AFL++ is ready for that version. 

In the end it's just formatting, and I believe consistency is better than preference.

I've made a commit suggesting a way to approach this:
1. If CLANG_FORMAT_BIN is set _and_ exits, use that
2. else check if clang-format-$LLVM_VERSION exists (as in the container) and use that
3. else check if clang-format-CURRENT_LLVM exists (hardcoded in the .py) and use that
4. else complain that we need clang-format-$LLVM_VERSION (or hardcoded CURRENT_LLVM if that env doesn't exist as with most users))

WDYT?

Reply:
Probably it also makes sense to have the current `$LLVM_VERSION` somewhere in the repo in a single file, instead of multiple

Reply:
hmm I think a format checker is not a good idea. every submitter would need to use the same llvm and update when we command everyone does. I prefer that we do it ourselves from time to time.
so please disable this..